metadata-io/src/main/java/com/linkedin/metadata/entity/AspectUtils.java

```diff
@@ -39,6 +51,46 @@ public static List<MetadataChangeProposal> getAdditionalChanges(
         .collect(Collectors.toList());
   }
 
+  @Nullable
+  public static Aspect getLatestAspect(
```

Comment:
so this is just a helper method for `getV2`? Do we not have a helper method anywhere else?

Reply:
Removing this its unused!